modules/shortlinks.php

```diff
@@ -85,5 +85,11 @@ function wpme_get_shortlink( $id = 0, $context = 'post', $allow_slugs = true ) {
 }
 
 function wpme_get_shortlink_handler( $shortlink, $id, $context, $allow_slugs ) {
-	return wpme_get_shortlink( $id, $context, $allow_slugs );
+	if ( $shortlink === false ) {	// respect / do not change the value from previous filter hooks
+		$shortlink = wpme_get_shortlink( $id, $context, $allow_slugs );	// returns wp.me shortlink or empty string
+		if ( $shortlink === '' ) {	// empty string for empty post object or no post type support for shortlinks
```

Comment:
Could you use Yoda conditions here as well?

Reply:
Done as well. ;-)